### DISCUSSION TITLE
Remove github pages.

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [collab-community/journey-book#19] [General]
You setup github pages @Adarsh-jaiss , I think it's coz of #1 , I think you should add that only when we begin working on that lol. Also, I suggest you use Netlify or Vercel to deploy (preferably Netlify) because, when a contributor makes a PR relating to the website, it will display a deploy preview, which makes it easier for everyone, so you won't have to always take it to localhost or anything.

Also, I can try to get a domain for ya if we work on #1 if required, unless you are part of the GitHub Student Plan.

**Comment by Adarsh-jaiss** [comment]
-- okay, let me see how can I do that :)

**Reply by Adarsh-jaiss** [reply]
it is already disbaled.

**Reply by Adarsh-jaiss** [reply]
![github pages](https://user-images.githubusercontent.com/96974600/157456571-28420d5b-d2ae-4f3d-a4a6-226a4ba669fd.png)

**Reply by Adarsh-jaiss** [reply]
also disabled it from github actions :)

**Reply by Adarsh-jaiss** [reply]
and deleted it :)

**Reply by KendallDoesCoding** [reply]
Yup, thanks a lot!

**Comment by KendallDoesCoding** [comment]
Funny how we started it off with GitHub Pages, then went to Netlify now coming back to GitHub Pages, sorry for the detour @Adarsh-jaiss .

**Reply by Adarsh-jaiss** [reply]
Hahaha 😄😄 it's fine. It's actually feels like a game

**Reply by KendallDoesCoding** [reply]
lol
